src/components/HSButton/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface HSButtonProps {
+  label: string;
+  type?: ButtonVariant;
+  color?: ButtonColor;
```

Comment:
I think we should only limit the color to "success", "warning", and "danger". Then call the prop "state" to be in line with the [button specifications](https://hashi-docs.netlify.app/docs/next/design/button#states) in the docs.

Reply:
Done this change